src/simulators/matrix_product_state/svd.cpp

```diff
@@ -552,9 +553,23 @@ status csvd(cmatrix_t &A, cmatrix_t &U, rvector_t &S, cmatrix_t &V) {
 void csvd_wrapper(cmatrix_t &A, cmatrix_t &U, rvector_t &S, cmatrix_t &V,
                   bool lapack) {
   if (lapack) {
+
+#ifdef AER_THRUST_CUDA
+    cutensor_csvd_wrapper(A, U, S, V);
+#endif // AER_THRUST_CUDA
+
+#ifndef AER_THRUST_CUDA
     lapack_csvd_wrapper(A, U, S, V);
+#endif // AER_THRUST_CUDA
   } else {
+
+#ifdef AER_THRUST_CUDA
+    cutensor_csvd_wrapper(A, U, S, V);
+#endif // AER_THRUST_CUDA
+
+#ifndef AER_THRUST_CUDA
     qiskit_csvd_wrapper(A, U, S, V);
+#endif // AER_THRUST_CUDA
```

Comment:
Could you add code to select calling GPU version or CPU version by referring to option `device=GPU/CPU` even if building with `AER_THRUST_BACKEND=CUDA`

Reply:
Done! :)